docker/nats-sub/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM golang:1
 MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+ENV CMSMON_TAG=go-0.0.0
 ENV WDIR=/data
 WORKDIR $WDIR
-RUN curl -ksLO https://raw.githubusercontent.com/dmwm/CMSMonitoring/master/src/go/NATS/nats-sub.go
-RUN go mod init nats-sub.go
-RUN go get github.com/nats-io/nats.go
-RUN go mod tidy
-RUN go build nats-sub.go
+RUN curl -ksLO https://raw.githubusercontent.com/dmwm/CMSMonitoring/${CMSMON_TAG}/src/go/NATS/nats-sub.go && \
+    curl -ksLO https://raw.githubusercontent.com/dmwm/CMSMonitoring/${CMSMON_TAG}/src/go/NATS/go.mod &&  \
```

Comment:
Thanks @vkuznet , I though that getting exactly the same versions with the git tag could be a good fit for versioning but no need it as you suggested, since they will be build at the same time. Tested and applied to PR.